Makefile

```diff
@@ -0,0 +1,75 @@
+#Python3 and virtual environment
+PY3 := $(shell command -v python3 2> /dev/null)
+VENV := .venv
```

Comment:
This is conceptually unsound. re-using the virtual environment will lead to issues when/if dependencies change, especially if versions of sub-dependencies change.

For a test to be "clean" the venv should be re-initialized each time.

So, it's better to use `VENV := $(mktemp -d)`